bag/search/views.py

```diff
@@ -318,12 +317,16 @@ def _get_url(request, hit):
     if detail_type not in _details:
         raise ValueError('Cannot create self url %s %s', doc_type, hit.subtype)
 
-    if hit.subtype in ['gemeente']:
-        subtype_id = hit.naam
```

Comment:
Deze voorwaarde komt te vervallen?

Reply:
subtype_id werd niet gebruikt . Is overblijfsel van vroeger